mesa/batchrunner.py

```diff
@@ -129,27 +128,57 @@ def _make_model_args(self):
         all_kwargs = []
         all_param_values = []
 
-        count = len(self.parameters_list)
-        if count:
+        _count = len(self.parameters_list)
```

Comment:
I added that just based on what my IDE (pycharm) was telling me acceptable convention. My understanding was because it was for the function parameter. I am good if it is there or not.

Reply:
fixed